android-sdk/src/main/java/com/mobilecoin/lib/Verifier.java

```diff
@@ -12,73 +12,44 @@
  * Verifier is used for attestation connection verification.
  */
 
-public final class Verifier extends Native {
-    private static final String TAG = Verifier.class.getName();
+public final class Verifier {
```

Comment:
The `Verifier` class is left in place but the underlying bindings are removed. The public API for the `Verifier` class is unaffected, but underneath, the `Verifier` class now uses the `TrustedIdentities` API. This prevents apps from requiring code changes immediately on upgrading. The old API still works as it did before, but it is marked as deprecated.